DefineOptions.cmake

```diff
@@ -11,6 +11,7 @@ option(USE_UNITY "Use libunity to report progress in the launcher" OFF)
 option(USE_OPENMP "Use openmp threading support." ON)
 option(USE_OPENCL "Use OpenCL support." ON)
 option(USE_GRAPHICSMAGICK "Use GraphicsMagick library for image import." ON)
+option(USE_IMAGEMAGICK "Use ImageMagick library for image import." ON)
```

Comment:
Please set this to OFF for now as discussed (or again something not clear on my side:). For now devs needed it will activate this option.